docs/modules/ROOT/pages/documentation/ecosystem/docker-micro-usage.adoc

```diff
@@ -0,0 +1,165 @@
+== Usage
+
+=== Quick start
+
+To start the docker container and run Payara Micro:
+
+....
+docker run -p 8080:8080 payara/micro
+....
+
+It runs Payara Micro without any applications, therefore accessing the HTTP server bound to port 8080 will just return HTTP code 404 - Not Found. The Payara Micro process runs under the `payara` user.
+
+You need to add your applications to the container and deploy them.
+
+It is always recommended to specify the tag name of the Docker Image that you like to run. Otherwise, you are running the latest version and might encounter some unwanted effects due to changes between versions.
+
+....
+docker run -p 8080:8080 payara/micro:{currentVersion}
+....
+
+
+=== JDK Version
+
+_Since Payara 5.2020.3_
```

Comment:
```suggestion
```
JDK 11 variants are supported for all versions that are published in Nexus (since 5.201 but there are no Docker images before 5.201). So we can remove the _Since_ line

Reply:
Seems PR didn't include my latest changes locally. It have 5.201 locally, but I'll remove